driver/razerkbd_driver.c

```diff
@@ -2001,6 +2013,7 @@ static ssize_t razer_attr_write_matrix_effect_static(struct device *dev, struct
     case USB_DEVICE_ID_RAZER_BLACKWIDOW_V3_MINI:
     case USB_DEVICE_ID_RAZER_BLADE_15_BASE_2022:
     case USB_DEVICE_ID_RAZER_TARTARUS_V2:
+//    case USB_DEVICE_ID_RAZER_BLACKWIDOW_V4_X:
```

Comment:
If this is not needed, please remove the commented line.

Reply:
It's now removed